Missing file ?

I cannot run build since the `itch.h` includes the `endian.h` which does not exist:

```
$ sh build.sh 
In file included from main.cpp:6:
./itch.h:2:10: fatal error: 'endian.h' file not found
#include <endian.h>
         ^~~~~~~~~~
1 error generated.
In file included from main.cpp:6:
./itch.h:2:10: fatal error: 'endian.h' file not found
#include <endian.h>
         ^~~~~~~~~~
1 error generated.
```
 I'm on MacOs Mojave 10.14.4. 
```
$ clang --version
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.5.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

I have googled the issue but [this](https://stackoverflow.com/questions/27073837/fatal-error-endian-h-file-not-found) does not help.

Thanks.

--- a/itch.h
+++ b/itch.h
@@ -1,5 +1,5 @@
 #pragma once
-#include <endian.h>
+#include <sys/types.h>
 
 enum class itch_t {
   SYSEVENT = 'S',

